packages/owc-dev-server/owc-dev-server.js

```diff
@@ -126,26 +33,92 @@ if (modulesDir) {
 
   transformOptions.modulesUrl = modulesDir;
 }
+
+/** Serves app index.html, injecting watch mode script if watching */
+function serveIndexHTML(req, res, next) {
+  if ((req.method === 'GET' || req.method === 'HEAD') && req.accepts('html')) {
+    let index = fs.readFileSync(path.join(process.cwd(), path.sep, appIndex), 'utf-8');
+    if (watch) {
+      index = index.replace('</body>', `${reloadInjectScript}</body>`);
+    }
+    res.send(index);
+  } else {
+    next();
+  }
+}
+
+/** Serves app index.html if the request was not a file request. This allows SPA routing */
+function serveIndexHTMLFallback(req, res, next) {
+  if (!req.url.includes('.')) {
+    serveIndexHTML(req, res, next);
+  } else {
+    next();
+  }
+}
+
+/** Adds any served files to file watcher, reloading the browser on change */
+if (watch) {
```

Comment:
Docs say that watch requires appIndex, so cli should exit with an error if it is not set

Reply:
Great point